src/main/java/com/verapdf/restapi/service/ConfigService.java

```diff
@@ -0,0 +1,13 @@
+package com.verapdf.restapi.service;
+
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class ConfigService{
+
+    String getSavePath() {
+        return "Jobs";
```

Comment:
this class is useless. We shall use configuration files (application.properties, for example) to make such configs